src/frontend/komponenter/Infotrygd/Infotrygd.tsx

```diff
@@ -0,0 +1,102 @@
+import React from 'react';
+
+import styled from 'styled-components';
+
+import { Knapp } from 'nav-frontend-knapper';
+import { Input, SkjemaGruppe } from 'nav-frontend-skjema';
+import { Innholdstittel } from 'nav-frontend-typografi';
+
+import { byggFunksjonellFeilRessurs, Ressurs, RessursStatus } from '@navikt/familie-typer';
+
+import { useInfotrygd } from '../../context/InfotrygdContext';
+import { IInfotrygdsaker } from '../../typer/infotrygd';
+import { hentFrontendFeilmelding } from '../../utils/ressursUtils';
+import { Sakstabell } from './Sakstabell';
+
+const InfotrygdContainer = styled.div`
```

Comment:
Litt blanding av rem og px her. Vi pleier å bruke rem i team familie, så supert om du får skrevet om :)

Reply:
Ok - fjerner til rem der px går opp i rem. (16px, 32px osv). De andre er for spesifikke.